pkg/network/ovn_kubernetes.go

```diff
@@ -634,6 +672,68 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	return objs, progressing, nil
 }
 
+// shouldRenderIPsec method ensures the have following IPsec states for upgrade path from 4.14 to 4.15 or later versions:
+// When 4.14 cluster is already installed with MachineConfig for IPsec extension and ipsecConfig: {} is set in networks operator
```

Comment:
replace `ipsecConfig: {} is set` with `ipsecConfig is set` throughout this text block